dotcom-rendering/src/components/Header.tsx

```diff
@@ -39,6 +39,7 @@ type Props = {
 	idApiUrl: string;
 	headerTopBarSearchCapiSwitch: boolean;
 	hasPageSkin?: boolean;
+	pageId?: string;
```

Comment:
Can this ever be `undefined`?

Reply:
The Header component is called in 10 other components and `<FrontLayout />` is the only one I've added this new prop to.